pom.xml

```diff
@@ -103,7 +103,7 @@
                             <serverId>ossrh</serverId>
                             <nexusUrl>https://oss.sonatype.org/</nexusUrl>
                             <autoReleaseAfterClose>true</autoReleaseAfterClose>
-                            <stagingProgressTimeoutMinutes>10</stagingProgressTimeoutMinutes>
+                            <stagingProgressTimeoutMinutes>20</stagingProgressTimeoutMinutes>
```

Comment:
Dont have knowledge as to why we need this change .. Can you explain a bit for my curiosity ?

Reply:
This was because the pr validation workflow (build, run test) was timing out due to the log4j changes causing the build  PR validation to fail.  Scott suggested to increase this timeout to avoid the failure https://twilio.slack.com/archives/C8AL9KPPH/p1639787570158300